tiled/server/authentication.py

```diff
@@ -233,6 +218,92 @@ async def get_session_state(decoded_access_token=Depends(get_decoded_access_toke
         return decoded_access_token.get("state")
 
 
+async def move_api_key(request: Request, api_key: Optional[str] = Depends(get_api_key)):
+    if ("api_key" in request.query_params) and (
+        request.cookies.get(API_KEY_COOKIE_NAME) != api_key
+    ):
+        request.state.cookies_to_set.append(
+            {"key": API_KEY_COOKIE_NAME, "value": api_key}
+        )
+
+
+async def get_scopes_from_api_key(
+    api_key: str, settings: Settings, authenticators, db
+) -> Sequence[str]:
+    if not authenticators:
+        # Tiled is in a "single user" mode with only one API key.
+        return (
+            {
```

Comment:
While we're here, I wonder if this set should be defined as a global variable, `SINGLE_USER_SCOPES`, to improve readability.

Reply:
Add a USER_SCOPES, not attached to the name if you think it's important to call out this is for the TILED_SINGLE_USER_API_KEY case.